libs/vertexai/langchain_google_vertexai/model_garden.py

```diff
@@ -70,3 +88,140 @@ async def _agenerate(
             endpoint=self.endpoint_path, instances=instances
         )
         return self._parse_response(response)
+
+
+class ChatAnthropicVertex(_VertexAICommon, BaseChatModel):
+    async_client: Any = None  #: :meta private:
+    model_name: str = "claude-3-sonnet@20240229"
+    "Underlying model name."
+    max_output_tokens: int = 1024
+
+    @root_validator()
+    def validate_environment(cls, values: Dict) -> Dict:
+        from anthropic import (  # type: ignore[import-not-found]
+            AnthropicVertex,
+            AsyncAnthropicVertex,
+        )
+
+        values["client"] = AnthropicVertex(
+            project_id=values["project"],
+            region=values["location"],
+            max_retries=values["max_retries"],
+        )
+        values["async_client"] = AsyncAnthropicVertex(
+            project_id=values["project"],
+            region=values["location"],
+            max_retries=values["max_retries"],
+        )
```

Comment:
I'd rather keep it here, since we might have other partner integrations on Vertex Model Garden eventually, so `langchain-google-vertexai` sounds like a more general place to me when people are looking for this integration. wdyt?

Reply:
Works for me. From what I can tell here: https://docs.anthropic.com/claude/reference/claude-on-vertex-ai

It looks like it requires both sdks